packs/feat-effects/stance-desert-wind.json

```
@@ -0,0 +1,36 @@
+{
+    "_id": "gcR66Xgi12ICOVt7",
+    "img": "icons/magic/perception/eye-slit-red-orange.webp",
+    "name": "Stance: Desert Wind",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Desert Wind]<br /><br /></p>\n<p>A vortex of sand and dust surrounds you. Each creature inside your kinetic aura (including you) is @UUID[Compendium.pf2e.conditionitems.Item.Concealed] from any creature outside your kinetic aura, and creatures outside the aura are concealed from creatures inside the aura other than you. Your air impulses carry sands that cut at great speed.</p>\n<p>When you use a damaging air impulse that affects at least one creature in your kinetic aura, that impulse deals 1 additional slashing damage, or 2 if the impulse has a single target.</p>\n<hr />\n<p><strong>Level (+2)</strong> The extra damage increases by 1, or 2 if the impulse has a single target.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Desert Wind]</p>\n<p>A vortex of sand and dust surrounds you. Each creature inside your kinetic aura (including you) is @UUID[Compendium.pf2e.conditionitems.Item.Concealed] from any creature outside your kinetic aura, and creatures outside the aura are concealed from creatures inside the aura other than you. Your air impulses carry sands that cut at great speed.</p>\n<p>When you use a damaging air impulse that affects at least one creature in your kinetic aura, that impulse deals 1 additional slashing damage, or 2 if the impulse has a single target.</p>\n<hr />\n<p><strong>Level (+2)</strong> The extra damage increases by 1, or 2 if the impulse has a single target.</p>"
```
The impulse damage part of this is doable, but it will be tricky. Maybe best done as a toggle with two suboptions. We can do that as a follow up though.